app/metrics.yaml

```diff
@@ -1331,7 +1350,7 @@ search_engines:
       - rtestard@mozilla.com
     expires: never
     extra_keys:
-      current_engines_count:
+      current_:
```

Comment:
@perrymcmanis144 is this intentional?

Reply:
I don't have this in my local...

ok let me fix